client/client.go

```diff
@@ -63,15 +63,15 @@ var ErrRepositoryNotExist = errors.New("repository does not exist")
 // NotaryRepository stores all the information needed to operate on a notary
 // repository.
 type NotaryRepository struct {
-	baseDir         string
-	gun             string
-	baseURL         string
-	tufRepoPath     string
-	fileStore       store.MetadataStore
-	CryptoService   signed.CryptoService
-	tufRepo         *tuf.Repo
-	roundTrip       http.RoundTripper
-	KeyStoreManager *keystoremanager.KeyStoreManager
+	baseDir       string
+	gun           string
+	baseURL       string
+	tufRepoPath   string
+	fileStore     store.MetadataStore
+	CryptoService signed.CryptoService
+	tufRepo       *tuf.Repo
+	roundTrip     http.RoundTripper
+	Manager       *certs.Manager
```

Comment:
Probably good to keep the field name as `CertManager` (sorry, I know you had that before my suggestion). Just reading further down, `Manager` is too generic for a field name, without the preceding `certs` package name it's a little obscure.